error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/ImplicitBlockingFluxOperationTest.java

```diff
@@ -0,0 +1,141 @@
+            "",
+            "class A {",
+            "  @SuppressWarnings(\"ImplicitBlockingFluxOperation\")",
+            "  void m() {",
+            "    flux().toIterable();",
+            "    flux().toStream();",
+            "  }",
+            "",
+            "  Flux<Integer> flux() {",
+            "    return Flux.just(1, 2, 3);",
+            "  }",
+            "}")
+        .doTest(BugCheckerRefactoringTestHelper.TestMode.TEXT_MATCH);
+  }
+
+  @Test
+  void replacementSecondSuggestedFix() {
+    BugCheckerRefactoringTestHelper.newInstance(ImplicitBlockingFluxOperation.class, getClass())
+        .setFixChooser(FixChoosers.SECOND)
+        .addInputLines(
+            "A.java",
+            "import reactor.core.publisher.Flux;",
+            "",
+            "class A {",
+            "  void m() {",
+            "    flux().toIterable();",
+            "    flux().toStream();",
+            "  }",
+            "",
+            "  Flux<Integer> flux() {",
+            "    return Flux.just(1, 2, 3);",
+            "  }",
+            "}")
+        .addOutputLines(
+            "A.java",
+            "import static com.google.common.collect.ImmutableList.toImmutableList;",
+            "",
+            "import reactor.core.publisher.Flux;",
+            "",
+            "class A {",
+            "  void m() {",
+            "    flux().collect(toImmutableList()).block();",
+            "    flux().collect(toImmutableList()).block().stream();",
+            "  }",
+            "",
+            "  Flux<Integer> flux() {",
+            "    return Flux.just(1, 2, 3);",
+            "  }",
+            "}")
+        .doTest(BugCheckerRefactoringTestHelper.TestMode.TEXT_MATCH);
+  }
+
+  @Test
+  void replacementThirdSuggestedFix() {
+    BugCheckerRefactoringTestHelper.newInstance(ImplicitBlockingFluxOperation.class, getClass())
+        .setFixChooser(FixChoosers.THIRD)
+        .addInputLines(
+            "A.java",
+            "import reactor.core.publisher.Flux;",
+            "",
+            "class A {",
+            "  void m() {",
+            "    flux().toIterable();",
+            "    flux().toStream();",
+            "  }",
+            "",
+            "  Flux<Integer> flux() {",
+            "    return Flux.just(1, 2, 3);",
+            "  }",
+            "}")
+        .addOutputLines(
+            "A.java",
+            "import static java.util.stream.Collectors.toUnmodifiableList;",
+            "",
+            "import reactor.core.publisher.Flux;",
+            "",
+            "class A {",
+            "  void m() {",
+            "    flux().collect(toUnmodifiableList()).block();",
+            "    flux().collect(toUnmodifiableList()).block().stream();",
+            "  }",
+            "",
+            "  Flux<Integer> flux() {",
+            "    return Flux.just(1, 2, 3);",
+            "  }",
+            "}")
+        .doTest(BugCheckerRefactoringTestHelper.TestMode.TEXT_MATCH);
```

Comment:
```suggestion
        .doTest(TestMode.TEXT_MATCH);
```
This is a convention we have.

Reply:
Pushed this change :).